site/docs/ch/meta-service/table-management.md

```diff
@@ -43,6 +45,16 @@ Logstore 作为可选组件则需要另外的配置来指明，有关 Logstore 
 上面的例子中将 kafka 集群：`127.0.0.1:9092`中的 topic：`local_catalog.test_db.test_log_store.log_store`作为了新建表的 Logstore,
 而在执行上面的语句之前你还需要手动去对应的 Kafka 集群中创建对应的 topic，或者打开集群的自动创建 topic 功能。
 
+### 配置 Watermark
+水位（watermark） 用于表述表的写入进度，具体来说它是表上的一个类型为时间戳的属性，意为小于这个时间戳的数据都已经写入到表中，它一般用来观察表的写入进度，也可以作为下游批量计算任务的触发指标。
+
+上面的例子中将 `op_time` 设置为表的事件时间字段，在数据写入时会用写入数据的 `op_time` 来计算表的水位，同时为了应对写入乱序的问题，设置在计算水位时允许的数据迟到时间为一分钟。
+可以在 [AMS Dashboard](http://localhost:1630) 的表详情中看到表当前的水位，也可以在`Terminal`中输入下面的 SQL 来查询表的水位：
+
+```sql
+SHOW TBLPROPERTIES test_db.test_log_store ('table.watermark');
```

Comment:
Maybe showing the result of the table watermark in the Documentation is more intuitive and friendly for clients.

Reply:
Added result part.